src/main/resources/data-repository-openapi.yaml

```diff
@@ -254,11 +323,46 @@ definitions:
 
   StudyModel:
     description: |
-      Complete description of a study.
+      Complete definition of a study.
     type: object
     properties:
       id:
         $ref: '#/definitions/UniqueIdModel'
+      name:
+        $ref: '#/definitions/DatasetNameModel'
+      description:
+        type: string
+        description: Description of the study
+      default_dataset_id:
+        type: string
+        description: Id of the auto-generated default passthru dataset
+      custodians:
+        type: string
+        description: list of custodian (emails?)
```

Comment:
i think we should represent lists like this (and i suggested a different description):
```
type: array
items:
  type: string
description: Emails of users and groups who should have custodian access to this study.
```

Reply:
sounds good